### PR TITLE
Adapt to support ESB v1.0.*

### PR DESCRIPTION
**Pull request by trandangtri**
- Using `FieldSort`, instead of `Sort` object
- Replace `ONGR Strick Standard` by `PSR2`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,22 +1,12 @@
...
-    - php: hhvm
-    - php: 7.0
-
 before_script:
   - composer install --no-interaction
-
 script:
   - vendor/bin/phpunit --coverage-clover=coverage.clover
-  - vendor/bin/phpcs -p --standard=vendor/ongr/ongr-strict-standard/Ongr --ignore=vendor/ ./
+  - vendor/bin/phpcs -p --standard=PSR2 --ignore=vendor/,Tests/app/ ./
```
Comment (saimaz) [inline .travis.yml]: There is no such a directory `Tests/app/`. Please remove it.

File: .travis.yml [hunk 2]
```diff
@@ -1,22 +1,12 @@
...
 php:
-  - 5.4
   - 5.5
   - 5.6
   - 7.0
   - hhvm
-
-matrix:
-  allow_failures:
-    - php: hhvm
```
Comment (saimaz) [inline .travis.yml]: I'm not so sure we want to keep up `hhvm` full support. It's nice to check compatibility but IMO we should allow it to fail.

File: docs/Aggregation/TopHits.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ so that the top matching documents can be aggregated per bucket.
 And now the query via DSL:
 
 ```php
-$sort = new Sort('last_activity_date', Sort::ORDER_DESC);
+$sort = new FieldSort('last_activity_date', Sort::DESC);
```
Comment (saimaz) [inline docs/Aggregation/TopHits.md]: I think it has to be actually `$sort = new FieldSort('last_activity_date', FieldSort::DESC);` No?

**Comment by trandangtri** [comment]
@saimaz, I updated. Thanks for your feedbacks.

**Pull request merged**